defect-location-server/backend/src/main/java/cn/com/devopsplus/dop/server/defect/config/PredictConfig.java

```diff
@@ -1,48 +1,27 @@
 package cn.com.devopsplus.dop.server.defect.config;
 
+import lombok.Getter;
 import org.springframework.stereotype.Component;
 
 @Component
+@Getter
 public class PredictConfig {
-    private final String projectDirectory="D:/";
-    private final String pythonInterpreterPath="D:/python/python.exe ";
-    private final String pythonProjectPath="D:/JITO/JITO-Identification";
-    private final String buildModelPythonPath=pythonProjectPath+"/defect_features/build_model.py ";
-    private final String predictModelPythonPath= pythonProjectPath+"/defect_features/run_model.py ";
-    private final String locationModelPythonPath= pythonProjectPath+"/defect_features/Corpus.py ";
-    private final String sbflPythonPath= pythonProjectPath+"/defect_features/SBFL/location.py ";
-    private final String sbflresultPath= pythonProjectPath+"/defect_features/SBFL/result3.csv";
+    private final String AlgorithmAddress="http://127.0.0.1:5000";
 
-    public String getSbflresultPath() {
-        return sbflresultPath;
-    }
+    private final String dataPath="D:/Fee/";
```

Comment:
This path should be in Linux FHS, not Windows filesystem. Paths below have the same problem.